debian/frr-pythontools.install

```diff
@@ -1 +1,2 @@
 usr/lib/frr/frr-reload.py
+tools/generate_support_bundle.py usr/lib/frr/
```

Comment:
Pretty sure this can just be `usr/lib/frr/generate_support_bundle.py`

Reply:
Yes, both formats works in this case. I thought this format is little more explicit and clear to whoever may be checking this code at later point. 

But, I can change the format as you had suggested.

Reply:
Yes it's more explicit, however the rest of the packaging stuff uses implicit paths where possible, I think consistency is better